packages/base-styles/_mixins.scss

```diff
@@ -454,6 +454,13 @@
 	// Darker shades.
 	--wp-admin-theme-color-darker-10: #{darken($color-primary, 5%)};
 	--wp-admin-theme-color-darker-20: #{darken($color-primary, 10%)};
+
+	// Focus style width.
```

Comment:
@gziolo or @aristath if either of you have time, is this the right place to register this rule? I placed it here because then it's in context of the other CSS variables registered. But the side-effect is that the rule is duplicated multiple times in the DOM (something to look at separately for the color, perhaps, but perhaps best to not exacerbate with this PR). Is there a more appropriate place to store this rule? The base block-list CSS perhaps? Thank you for looking!

Reply:
Yes, it's a bit unfortunate that it gets duplicated. There is an issue opened by @aristath: https://github.com/WordPress/gutenberg/issues/27663 that track that behavior. I guess it's fine to land this PR and tackle code duplication separately 👍🏻

Reply:
Can I get approval? :)